guides/client/uploads-external.md

```diff
@@ -1,4 +1,4 @@
-# External Uploads
+# Uploads (External)
```

Comment:
:+1: 

Since part of the intent here is to update link text, may I suggest updating a couple of places that refer to this page

- https://github.com/phoenixframework/phoenix_live_view/blob/6648f774354e308efc26d7e44e6e68d5a1b12fe1/guides/client/js-interop.md?plain=1#L24-L26
- https://github.com/phoenixframework/phoenix_live_view/blob/6648f774354e308efc26d7e44e6e68d5a1b12fe1/guides/server/uploads.md?plain=1#L214-L215

Reply:
Nice catch. 
Updated.

Reply:
It might just be me, but I find "Uploads (External)" not as nice to read as "External Uploads"; so I'd suggest to change the one reference in the welcome guide to "External uploads" instead. Any opinions on that?

Reply:
Not just you, I had the same feeling. "Uploads (External)" would be somehow good when the items were sorted alphabetically to somehow fall next to and contrast to "Uploads" (the server-side ones).

+1 to "External uploads" (I think José has a preference on not using "Title Case", see https://elixirforum.com/t/proposal-for-consistent-capitalization-in-documentation-titles/61636/2?u=rhcarvalho)

Reply:
Updated. "External uploads" is used.